watertap/unit_models/ion_exchange_0D.py

```diff
@@ -242,20 +365,27 @@ def build(self):
         # Wilson and Geankoplis, Ind. Eng. Chem. Fundam., 5, 9 (1966)
 
         self.Sh_A = Param(
-            initialize=1.09,
+            initialize=2.4,
```

Comment:
Are these changed because of incorrect units in the past? Not clear

Reply:
I changed the regression I was using for Sh, that is why the number changed.